src/main/scala/rocketchip/Configs.scala

```diff
@@ -115,7 +115,10 @@ class BasePlatformConfig extends Config (
           idBits = Dump("MEM_ID_BITS", site(MIFTagBits)))
       }
       case BuildCoreplex => (p: Parameters) => Module(new DefaultCoreplex(p))
-      case NExtInterrupts => 2
+      case NExtTopInterrupts => 2
+      case NExtPeripheryInterrupts => 0
```

Comment:
I think it makes more sense to declare this inside the DeviceBlock class, since that's where the interrupts will be driven from.